src/test/java/sk/uniba/fmph/dcs/player_board/PlayerBoardTest.java

```diff
@@ -0,0 +1,210 @@
+package sk.uniba.fmph.dcs.player_board;
+
+import org.json.JSONObject;
+import sk.uniba.fmph.dcs.stone_age.Effect;
+import sk.uniba.fmph.dcs.stone_age.EndOfGameEffect;
+import org.junit.Test;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+public class PlayerBoardTest {
+    @Test
+    public void testAddPoints() {
+        PlayerBoard playerBoard = new PlayerBoard();
+        playerBoard.addPoints(10);
+        playerBoard.addPoints(5);
+
+        JSONObject state = new JSONObject(playerBoard.state());
+        assertEquals("15", state.getJSONObject("points").toString());
```

Comment:
```suggestion
        assertEquals("15", state.getString("points"));
```
Toto nebude fungovať, treba použiť `getString` všade, kde je `getJSONObject`.